<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>territorialFundPage</class>
 <widget class="QWidget" name="territorialFundPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Страховая принадлежность</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QCheckBox" name="chkTFCheckPolicy">
     <property name="text">
      <string>Разрешить использовать cервис поиска и проверки полиса ОМС</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblTFCPUrl">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Адрес сервиса</string>
     </property>
     <property name="buddy">
      <cstring>edtTFCPUrl</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>185</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>160</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnTFCPTest">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Проверить соединение</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="edtTFCPUrl">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;В случае унифицированного протокола можно вписать&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;http://${dbServerName}/ident/handler.php&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;При этом вместо &lt;span style=&quot; font-weight:600;&quot;&gt;${dbServerName} &lt;/span&gt;должно подставиться имя (или IP адрес) сервера базы данных.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QCheckBox" name="chkSyncAttachmentsAfterSave">
     <property name="text">
      <string>Синхронизировать прикрепление после сохранения рег.карты</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkTFCheckPolicy</tabstop>
  <tabstop>edtTFCPUrl</tabstop>
  <tabstop>btnTFCPTest</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chkTFCheckPolicy</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblTFCPUrl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkTFCheckPolicy</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtTFCPUrl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkTFCheckPolicy</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnTFCPTest</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
